modules/security-hardening-what.adoc

```diff
@@ -10,8 +10,7 @@ The link:https://docs.fedoraproject.org/en-US/Fedora/19/html/Security_Guide/chap
 {op-system-base} system.
 endif::[]
 ifdef::openshift-enterprise,openshift-webscale,openshift-aro[]
-The link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index#scanning-container-and-container-images-for-vulnerabilities_scanning-the-system-for-security-compliance-and-vulnerabilities[{op-system-base} 8 Security Hardening] guide describes how you should approach security for any
-{op-system-base} system.
+The link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index#scanning-container-and-container-images-for-vulnerabilities_scanning-the-system-for-security-compliance-and-vulnerabilities[{op-system-base} 8 Security Hardening] guide describes how you should approach security for any {op-system-base} system.
```

Comment:
May I know why not use Rhel9 link instead? Didn't remember clearly for ocp 4.12. But OCP 4.13 and above are based on rhel9. 
https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/security_hardening/index#Post-installation_procedures_securing-rhel-during-installation

Reply:
Thanks, @xiaojiey . It was out of scope but I made the change as the Security Hardening doc is closely relate to the target change.